lib/dest/write-contents/index.js

```diff
@@ -13,7 +13,7 @@ function writeContents(optResolver) {
 
   function writeFile(file, enc, callback) {
     // Write it as a symlink
-    if (file.symlink) {
+    if (file.isSymbolic()) {
```

Comment:
Not sure if the documents talk about `.symlink` but if so, it'll need to be updated